### ISSUE TITLE
Option to show full discography list on an Artist page

### ISSUE THREAD

**Issue by rappo** [jeffvli/feishin#447]
## What do you want to be added?

When viewing an Artist page, the "Recent releases" section is paginated and limited to one row.

It would be nice to have an option to "Show all releases" and have the tiles wrap to additional lines, if needed.

![image](https://github.com/jeffvli/feishin/assets/2245234/4a3aa438-8495-4a17-b8d1-13c37239c77d)

**Comment by Cambi0nn** [comment]
I agree with the above. Now I keep clicking on the discography, having a needless extra click.

Furthermore, the recent releases seems to follow the "recently added" instead of "release year/date", which is a bit odd. After all, a recent release is the one that released the most recent, not the one added the last. I'd also argue that release date is more interesting on an artist page anyways.
